Devops Content Update (#1454)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/120-awk.md

> # Awk
> Awk is a general-purpose scripting language used for manipulating data or text and generating reports in linux world.
> 
> It is mostly used for pattern scanning and processing. It searches one or more files to see if they contain lines that matches with the specified patterns and then perform the associated actions. 
> 
> **Syntax** : 
> 
> `$ awk options 'selection_criteria {action}' input-file > output-file`
> 
> e.g. *$ awk '{print}' file.txt*
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/awk-command-unixlinux-examples/'>What is AWK? How to use it?</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://linuxize.com/post/awk-command/'>How AWK works?</BadgeLink>

Reviewer: ```suggestion
# awk

`awk` is a general-purpose scripting language used for manipulating data or text and generating reports in the Linux world. It is mostly used for pattern scanning and processing. It searches one or more files to see if they contain lines that match the specified patterns and then performs the associated actions. 

It has the below syntax: 

`awk options 'selection_criteria {action}' input-file > output-file` e.g. `$ awk '{print}' file.txt`

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/awk-command-unixlinux-examples/'>What is AWK? How to use it?</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://linuxize.com/post/awk-command/'>How AWK works?</BadgeLink>
```
